frontend_new/src/components/ThemeSetting.tsx

```diff
@@ -0,0 +1,16 @@
+
+function getDefaultTheme() {
+    //get the current date
+    var date = new Date();
+    
+    //0 to 23
+    var hour =  date.getHours();
+
+    if (hour < 6 || hour >22) { 
+        return false; 
+    } else {
+        return true; 
+    }
```

Comment:
**[Desired,Maintainability]** You could simplify this a bit, which would make it easier to maintain:
```javascript
let hour = new Date().getHours();
return hour < 6 || hour > 22;
```

Reply:
Changed it.